lib/aws/eks.ts

```diff
@@ -408,139 +408,167 @@ export class EksStack {
-              "eks.amazonaws.com/role-arn": hyperswitchServiceAccountRole.roleArn,
+        "hyperswitch-app": {
+          loadBalancer: {
+            targetSecurityGroup: lbSecurityGroup.securityGroupId
+          },
+          services: {
+            router: {
+              version: "v1.107.0",
+              image: `juspaydotin/hyperswitch-router:v1.107.0-standalone`,
+              host: "http://localhost:8080"
             },
-            server_base_url: "https://sandbox.hyperswitch.io",
-            secrets: {
-              podAnnotations: {
-                traffic_sidecar_istio_io_excludeOutboundIPRanges:
-                  "10.23.6.12/32",
-              },
-              kms_admin_api_key: kmsSecrets.kms_admin_api_key,
-              kms_jwt_secret: kmsSecrets.kms_jwt_secret,
-              kms_jwekey_locker_identifier1: kmsSecrets.kms_jwekey_locker_identifier1,
-              kms_jwekey_locker_identifier2: kmsSecrets.kms_jwekey_locker_identifier2,
-              kms_jwekey_locker_encryption_key1: kmsSecrets.kms_jwekey_locker_encryption_key1,
-              kms_jwekey_locker_encryption_key2: kmsSecrets.kms_jwekey_locker_encryption_key2,
-              kms_jwekey_locker_decryption_key1: kmsSecrets.kms_jwekey_locker_decryption_key1,
-              kms_jwekey_locker_decryption_key2: kmsSecrets.kms_jwekey_locker_decryption_key2,
-              kms_jwekey_vault_encryption_key: locker?.locker_ec2.locker_pair.public_key || kmsSecrets.kms_jwekey_vault_encryption_key,
-              kms_jwekey_vault_private_key: locker?.locker_ec2.tenant.private_key || kmsSecrets.kms_jwekey_vault_private_key,
-              kms_jwekey_tunnel_private_key: kmsSecrets.kms_jwekey_tunnel_private_key,
-              kms_jwekey_rust_locker_encryption_key: kmsSecrets.kms_jwekey_rust_locker_encryption_key,
-              kms_connector_onboarding_paypal_client_id: kmsSecrets.kms_connector_onboarding_paypal_client_id,
-              kms_connector_onboarding_paypal_client_secret: kmsSecrets.kms_connector_onboarding_paypal_client_secret,
-              kms_connector_onboarding_paypal_partner_id: kmsSecrets.kms_connector_onboarding_paypal_partner_id,
-              kms_key_id: kms_key.keyId,
-              kms_key_region: kms_key.stack.region,
-              kms_encrypted_api_hash_key: kmsSecrets.kms_encrypted_api_hash_key,
-              admin_api_key: admin_api_key,
-              jwt_secret: "test_admin",
-              recon_admin_api_key: "test_admin",
+            producer: {
+              image: `juspaydotin/hyperswitch-producer:v1.107.0-standalone`
             },
-            master_enc_key: kmsSecrets.kms_encrypted_master_key,
-            locker: {
-              locker_readonly_key: locker ? locker.locker_ec2.locker_pair.public_key : "locker-key",
-              hyperswitch_private_key: locker ? locker.locker_ec2.tenant.private_key : "locker-key",
+            consumer: {
+              image: `juspaydotin/hyperswitch-consumer:v1.107.0-standalone`
```

Comment:
Same here as well.

Reply:
no  juspaydotin/hyperswitch-router:v1.107.0 will have email feature enabled which is not supported in cdk script for now, so we need to use standalone image and enable kms via env params